### PR TITLE
Add attachment meta component

### PR DESCRIPTION
**Pull request by alex-ju**
Add an ["attachment meta" component](https://content-publisher-revie-pr-968.herokuapp.com/component-guide/attachment_meta) in a similar fashion with the ["image meta" component](https://content-publisher-revie-pr-968.herokuapp.com/component-guide/image_meta)

[Trello card](https://trello.com/c/z1aiStcD)

### CODE REVIEW

File: app/views/components/_attachment_meta.html.erb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<%
+  id ||= "attachment-meta-#{SecureRandom.hex(4)}"
```
Comment (kevindew) [inline app/views/components/_attachment_meta.html.erb]: Do we use this for anything?

File: app/views/components/docs/attachment_meta.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  * have alt text
+examples:
+  default:
+    data:
+      url: https://www.gov.uk/government/publications/minimum-wage-jobs-in-ashfield-local-authority-since-2009
+      thumbnail_src: https://assets.digital.cabinet-office.gov.uk/government/assets/pub-cover-doc-afe3b0a8a9511beeca890340170aee8b5649413f948e512c9b8ce432d8513d32.png
+      title: Document attachment
+      metadata_items:
+       - file_format: MS Word Document
```
Comment (kevindew) [inline app/views/components/docs/attachment_meta.yml]: We're going to hit something tricky with this and govspeak.

At the moment the information on the various texts to display here is stored in govspeak: https://github.com/alphagov/govspeak/blob/master/lib/govspeak/presenters/attachment_presenter.rb#L203-L240 (presumably Whitehall has a similar thing).

I don't really know what the right answer is for where to put this is.

This is a bit outside the scope of your card, just mentioning it as I see a problem looming which may make it harder to use this component 🤔

File: app/views/components/_attachment_meta.html.erb (hunk 1 above)
Comment (alex-ju) [reply app/views/components/_attachment_meta.html.erb]: Nope, not much. I thought would be good to have some consistency about the attributes that can be passed to a component.

File: app/views/components/docs/attachment_meta.yml (hunk 2 above)
Comment (benthorner) [reply app/views/components/docs/attachment_meta.yml]: Could we render the attachment using Govspeak and append the actions separately?

**Pull request merged**